.github/workflows/main-ci.yml

```
@@ -0,0 +1,280 @@
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+
+      - name: Install dependencies
+        run: npm ci || true
+
+      - name: Lighthouse CI
+        uses: treosh/lighthouse-ci-action@v10
```

actionlint (1.7.10)

[error] the runner of "treosh/lighthouse-ci-action@v10" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)